R/assess_has_bug_reports_url.R

```diff
@@ -0,0 +1,27 @@
+#' Assess a package for the presence of a url field where bugs can be reported.
+#'
+#' @eval roxygen_assess_family("has_bug_reports_url", 
+#' "a character value containing the BugReports field contents")
+#'
+#' @export
+assess_has_bug_reports_url <- function(x, ...){
+  has_bug_reports_url <- x$bug_reports_url
+  pkg_metric(has_bug_reports_url, class = "pkg_metric_has_bug_reports_url") 
+}
+
+# assign a friendly name for assess column
+attr(assess_has_bug_reports_url,"column_name") <- "has_bug_reports_url"
+attr(assess_has_bug_reports_url,"label") <- "presence of a bug_reports_url in repository"
```

Comment:
Very minor suggestion, but I think labels are better to be plain-english instead of referring to code objects

```suggestion
attr(assess_has_bug_reports_url,"label") <- "presence of a bug reports url in repository"
```